CharlieBackend.AdminPanel/Services/ScheduleService.cs

```diff
@@ -74,8 +75,8 @@ public async Task<EventOccurrenceDTO> GetEventOccurrenceById(long id)
 
         public async Task CreateScheduleAsync(CreateScheduleDto scheduleDTO)
         {
-              await _apiUtil
-                .CreateAsync<CreateScheduleDto>(_scheduleApiEndpoints.AddEventOccurrence, scheduleDTO);
+            await _apiUtil
+              .CreateAsync<EventOccurrenceDTO, CreateScheduleDto>(_scheduleApiEndpoints.AddEventOccurrence, scheduleDTO);
```

Comment:
Why do we call another method that returns the EventOccurrenceDTO object if we do not use it?